use_cases/eluc/experiments/predictor_experiments.ipynb

```diff
@@ -170,28 +181,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
     "forest_config = {\n",
+    "    \"features\": constants.NN_FEATS,\n",
     "    \"n_jobs\": -1,\n",
     "    \"max_features\": \"sqrt\",\n",
-    "    \"random_state\": 42\n",
+    "    \"random_state\": 42,\n",
     "}\n",
-    "forest = RandomForestPredictor(features=constants.NN_FEATS, **forest_config)"
+    "forest = RandomForestPredictor(**forest_config)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Note: The original paper trains from 1982 onwards but this is too slow and large for the\n",
     "# purpose of this example.\n",
-    "forest.fit(dataset.train_df.loc[2002:][constants.NN_FEATS], dataset.train_df.loc[2002:][\"ELUC\"])\n",
-    "forest.save(\"predictors/sklearn/trained_models/experiment_rf\")"
+    "forest_year = 1982\n",
```

Comment:
In the comment above you say training from 1982 "is too slow and large for the purpose of this example". But now you do it. Should you remove the comment?